### ISSUE TITLE
Deeper network

### ISSUE THREAD

**Issue by nassarofficial** [ZFTurbo/ZF_UNET_224_Pretrained_Model#2]
I know this is not an issue, but I wanted to contact you to know how did you make the network deeper in keras for the DSTL competition using this model?

**Comment by ZFTurbo** [comment]
I didn't really understand your question. Comparing to original UNET it has higher number of levels and larger input size.

**Comment by nassarofficial** [comment]
In the DSTL competition post u posted, the diagram of the network was much deeper (https://www.kaggle.com/c/dstl-satellite-imagery-feature-detection/discussion/29829) how did u pretrain it using this network?

**Comment by ZFTurbo** [comment]
It's actually the same network. You can print model.summary() to ensure.

Weights provided here wasn't obtained on the DSTL data but on infinite generator of random images. In DSTL competition I trained this network from the scratch.

**Comment by nassarofficial** [comment]
Thank you very much, one last question, how many epocs did it take you to reach this accuracy?!

**Comment by ZFTurbo** [comment]
Number of epochs depends on Learning rate and optimizer. Typically it was around 30-40 epochs for single model in DSTL competition.

**Comment by nassarofficial** [comment]
Thanks so uch for ur help!
